wasm/Writer.cpp

```diff
@@ -1521,7 +1564,7 @@ void Writer::run(bool undefinedEntry) {
         break;
      }
   }
-  if (hasCalls) {
+  if (hasCalls && symtab->syncCallEntryIsUndefined) {
```

Comment:
Should this be `!` ?

Reply:
`symtab->syncCallEntryIsUndefine` is true when no sync call entry point is provided mannually. That's why there is not need of `!`.

Reply:
https://github.com/AntelopeIO/cdt-lld/pull/2/commits/e59ac252d3caf05c5da2026a91abf11da44b0156